File: src/data/snippets/functions/snippet.js
// Function to calculate the area of a circle
function calculateCircleArea(radius) {
    // Area formula using Math.pow()
    return Math.PI * Math.pow(radius, 2);  
}

let radius = 5.0;  // Circle radius
let area = calculateCircleArea(radius);

// Print area with two decimal places
console.log(`Area: ${area.toFixed(2)}`);

File: src/data/snippets/functions/snippet.cs
using System;

// Function to calculate the area of a circle
double CalculateCircleArea(double radius) {
    // Area formula using Math.PI
    return Math.PI * (radius * radius);  
}

double radius = 5.0;  // Circle radius
double area = CalculateCircleArea(radius);

// Print area with two decimal places
Console.WriteLine($"Area: {area:F2}");
